contracts/StableJoeStaking.sol

```diff
@@ -134,7 +137,7 @@ contract StableJoeStaking is Initializable, OwnableUpgradeable {
 
         uint256 _fee;
         // Only EOAs holding Smol Joes are exempt from paying the deposit fee
-        if (smolJoes.balanceOf(_msgSender()) == 0 || _msgSender() != tx.origin) {
+        if ((address(smolJoes) != 0 && smolJoes.balanceOf(_msgSender()) == 0) || _msgSender() != tx.origin) {
```

Comment:
Wrong parenthesis, if smolJoe is not set EOAs wouldn't pay any fees, change to:
`if (address(smolJoes) != 0 && (smolJoes.balanceOf(_msgSender()) == 0 || _msgSender() != tx.origin)) {`